src/settings-ui/Microsoft.PowerToys.Settings.UI.Runner/OobeWindow.xaml.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation
```

Comment:
Let's structure the OOBE source code from the beginning to avoid tech debt later on.
Move all the OOBE code under a `OOBE` folder and also add the folder to the VS project.